src/chart/hawkular-chart-directive.ts

```diff
@@ -446,11 +447,11 @@ module Charts {
           function formatBucketedChartOutput(response) {
             //  The schema is different for bucketed output
             if (response) {
-
               return response.map((point:IChartDataPoint) => {
+                var timestamp = point.start + (point.start - point.end)/2;
```

Comment:
There is no point.timestamp with bucketed data, so before it would just render nothing at all.

You may need to handle bucketed data differently and use different logic when dealing with bucketed data (eg midpoint calculating) and when using non-bucketed data (eg just use point.timestamp since that data is available)